jans-auth-server/model/src/main/java/io/jans/as/model/jws/EDDSASigner.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Janssen Project software is available under the Apache License (2004). See http://www.apache.org/licenses/ for full text.
+ *
+ * Copyright (c) 2022, Janssen Project
+ */
+package io.jans.as.model.jws;
+
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.security.NoSuchProviderException;
+import java.security.Signature;
+import java.security.SignatureException;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.PKCS8EncodedKeySpec;
+import java.security.spec.X509EncodedKeySpec;
+
+import org.bouncycastle.jcajce.provider.asymmetric.edec.BCEdDSAPrivateKey;
+import org.bouncycastle.jcajce.provider.asymmetric.edec.BCEdDSAPublicKey;
+
+import io.jans.as.model.crypto.Certificate;
+import io.jans.as.model.crypto.signature.AlgorithmFamily;
+import io.jans.as.model.crypto.signature.EDDSAPrivateKey;
+import io.jans.as.model.crypto.signature.EDDSAPublicKey;
+import io.jans.as.model.crypto.signature.SignatureAlgorithm;
+import io.jans.as.model.util.Base64Util;
+
+/**
+ * Implementing the AbstractJwsSigner, that uses EDDSA for signing.
+ *
+ * @author Sergey Manoylo
+ * @version January 20, 2021
+ */
+public class EDDSASigner extends AbstractJwsSigner {
```

Comment:
`EDDSASigner` class is present but I don't see any usage of these class in entire PR. Maybe add test to see it in action.

Reply:
We have test for EDDSASigner, of course.
for example:

https://github.com/JanssenProject/jans/blob/jans-auth-server-extending-crypto-support-full/jans-auth-server/server/src/test/java/io/jans/as/server/comp/SignatureTest.java

https://github.com/JanssenProject/jans/blob/jans-auth-server-extending-crypto-support-full/jans-auth-server/server/src/test/java/io/jans/as/server/comp/JwtCrossCheckTest.java
.

This branch:
jans-auth-server-extending-crypto-support-full
contains fixed working (PASSED) tests in the JwtCrossCheckTest (where checking signings oxAuth <-> Nimbus is provided).

I hope, all this code will be moved to the main in next PRs.

Reply:
I see, ok, lets get this PR first and in next PR test for it. Not ideal but we have to move forward somehow.